doc/release-process.md

```diff
@@ -13,6 +13,11 @@ Before every minor and major release:
 * Update version in sources (see below)
 * Write release notes (see below)
 * Update `src/chainparams.cpp` nMinimumChainWork with information from the getblockchaininfo rpc.
+* Update `src/chainparams.cpp` defaultAssumeValid  with information from the getblockhash rpc.
+  - The selected value must not be orphaned so it may be useful to set the value two blocks back from the tip.
```

Comment:
> There is no reason to set it back at all except that if the block falls out of the chain the speedup will be lost.

Note we *could* make it smart enough to see the common parent block, but I'm not sure it's worth the additional effort.

> I also don't see a reason that this couldn't be updated at the last RC or even between an RC and a release. In general we should set it as far forward as possible...

If we're setting it last minute, I do think more than 2 blocks back would be best. Maybe 10.